SAMPLE_APPS.md

```diff
@@ -0,0 +1,83 @@
+# Try Out the Sample Apps
+
+Follow this guide to try out the sample app curated by Asgardeo. If you want to integrate Asgardeo for your own app, follow [this guide](/README.md#getting-started).
+
+
+## Table of Content
+
+-   [Create an Application in Asgardeo](#create-an-application-in-asgardeo)
+-   [Running the sample apps](#running-the-sample-apps)
+-   [Change the Application's Development Server Port](#change-the-applications-development-server-port)
+-   [Other Links](#other-links)
+
+
+## Create an Application in Asgardeo
+
+Before trying out the sample apps, you need to create an application in **Asgardeo**.
+
+1. Open the [**Asgardeo Console**](https://console.asgardeo.io/login) and go to **Applications** > **Develop** tab
+
+2. Click **New Application** and then **Single Page Application**.
+
+3. Enter any name you prefer as the name of the app
+
+4. Add the redirect URL(s). The redirect URL usually is the URL in which your React application is hosted. For local environments, it will be something like `https://localhost:3000`. This will be used as a configuration in [Running the sample apps](#2-running-the-sample-apps) section.<img width="1439" alt="Screenshot 2022-09-14 at 17 26 57" src="https://user-images.githubusercontent.com/42619922/190148189-bb933d6b-2f8e-41e7-8c42-9d67e6746d17.png">
```

Comment:
Will be good if we can make the following section a note after the step:

```
The redirect URL usually is the URL in which your React application is hosted. For local environments, it will be something like `https://localhost:3000`. This will be used as a configuration in [Running the sample apps](#2-running-the-sample-apps) section.<img width="1439" alt="Screenshot 2022-09-14 at 17 26 57" src="https://user-images.githubusercontent.com/42619922/190148189-bb933d6b-2f8e-41e7-8c42-9d67e6746d17.png">
```

Reply:
Fixed with 43f560a1043b10b9c8911b46908c6ee2a1d44c3d